src/yang/ietf-bgp-rib.yang

```diff
@@ -379,6 +392,27 @@ submodule ietf-bgp-rib {
           uses ext-community-attributes;
         }
       }
+      container large-communities {
+        description
+          "Enclosing container for the list of large community
+           attribute sets.";
+        list large-community {
+          key "index";
+          config false;
+          description
+            "List of path attributes that may be in use by multiple
+             routes in the table.";
+          leaf index {
+            type uint64;
+            description
+              "System generated index for each attribute set.  The
+               index is used to reference an attribute set from a
+               specific path.  Multiple paths may reference the same
```

Comment:
Two spaces after the period.

Reply:
That is already two spaces?

Reply:
Sorry, I meant to say there are two spaces. Did you want two spaces, or reduce it to one?

Reply:
I understand the question now.  I see an inconsistent set of spacing in the modules.  Tell me what the current trend is for style and I'll update this diff.  We'll get the rest as part of the cleanup issue.

Reply:
I try to leave one space between the period and the start of a new sentence. The RFC editor might do other edits, but that is out of my control.

Either ways, I am going to go ahead and approve this PR and we can cleanup afterwards.